03_controls/03_pairing/05_pairing_no_interaction.en.md

```diff
@@ -43,25 +43,27 @@ To check this control, the following resources may be useful:
 
 ## Example case
 
-We will use Wireshark with [BTVS](https://learn.microsoft.com/en-us/windows-hardware/drivers/bluetooth/testing-btp-tools-btvs) (btvs.exe -Mode wireshark) to capture and analyze the packets.
+In this example, a pairing test with wireless headphones is demonstrated.
 
-The Bluetooth device configuration tool of the laptop's operating system is used to initiate the connection. In Bluetooth Classic, the connection between the two devices starts with the _IO Capability Request_ command.
+For the test, Wireshark has been used with [BTVS](https://learn.microsoft.com/en-us/windows-hardware/drivers/bluetooth/testing-btp-tools-btvs) (btvs.exe -Mode wireshark) to capture and analyze packets. To perform the connection and pairing, the Bluetooth device configuration tool of the laptop's operating system is used.
 
-![Wireshark IO Caps Request]({{ '/assets/img/bsam-pa-01_io_cap_request.png' | relative_url}})
+The headphones use Bluetooth Classic, where most of the pairing process occurs in the controller and is not visible from the host. However, in this case, it is sufficient to observe the exchange of HCI messages, where the _IO Capability_ values ​​appear.
 
-The headphones respond with the _IO Capability Request Reply_ command with _IO Capability_ 0x03 (NoInputNoOutput). This allows pairings without user intervention.
+The first HCI messages of the pairing process between the two devices are the _IO Capability Request_ event, where the controller requests the local host's _IO Capabilities_, and the _IO Capability Request Reply_ command, where the host responds.
```

Comment:
It would be interesting to highlight that these refer to our own machine and not the device under test. For this reason, these should be ignored and not relevant to the outcome of the check.